executor/builder.go

```diff
@@ -130,6 +132,8 @@ func newExecutorBuilder(ctx sessionctx.Context, is infoschema.InfoSchema, ti *Te
 		isStaleness:      staleread.IsStmtStaleness(ctx),
 		txnScope:         txnManager.GetTxnScope(),
 		readReplicaScope: txnManager.GetReadReplicaScope(),
+		queryTS:          uint64(time.Now().UnixNano()),
+		localQueryID:     plannercore.AllocMPPQueryID(),
```

Comment:
Using `AllocMPPQueryID` in a non-mpp executor seems strange.
Can there be a more generic name?

Reply:
I'm not sure it's a good idea to hold `queryTS` and `queryTS` in ExecutorBuilder

Reply:
Even for non-mpp query, `plannercore.AllocMPPQueryID()` will be called?